Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m

```diff
@@ -164,12 +164,15 @@ - (void)configureLabelsWithConversationViewItem:(ConversationViewItem *)viewItem
 
     [self configureFonts];
 
+    NSString *timestampLabelText;
     if ([self isFailedOutgoingMessage:viewItem]) {
-        self.timestampLabel.text
+        timestampLabelText
             = NSLocalizedString(@"MESSAGE_STATUS_SEND_FAILED", @"Label indicating that a message failed to send.");
     } else {
-        self.timestampLabel.text = [DateUtil formatMessageTimestamp:viewItem.interaction.timestamp];
+        timestampLabelText = [DateUtil formatMessageTimestamp:viewItem.interaction.timestamp];
     }
+
+    self.timestampLabel.text = timestampLabelText.localizedUppercaseString;
```

Comment:
Hmm, there's a lot of places we're just using `uppercaseString`.  Can we please replace them with `localizedUppercaseString` - or make a card?

Reply:
I've changed to localizedUppercaseString everywhere it makes sense.